docs/conf.py

```diff
@@ -72,36 +80,31 @@
     "sphinx.ext.autodoc",
     "sphinx.ext.coverage",
     "sphinx.ext.doctest",
-    "sphinx.ext.inheritance_diagram",
     "sphinx.ext.intersphinx",
-    "sphinx.ext.mathjax",
-    "sphinx.ext.napoleon",
     "sphinx.ext.todo",
+    "sphinx.ext.inheritance_diagram",
+    "sphinx.ext.intersphinx",
     "sphinx.ext.viewcode",
+    "sphinx.ext.napoleon",
+    "sphinx.ext.mathjax",
 ]
+
+# Add any paths that contain templates here, relative to this directory.
+# templates_path = ["_templates"]
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
 html_extra_path = ["robots.txt"]
 exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
 source_suffix = ".rst"
 master_doc = "index"
-default_role = "obj"
-napoleon_use_rtype = False
-napoleon_google_docstring = False
-intersphinx_mapping = {
-    "astropy": ("https://docs.astropy.org/en/stable/", None),
-    "aiapy": ("https://aiapy.readthedocs.io/en/stable/", None),
-    "dask": ("https://docs.dask.org/en/latest", None),
-    "drms": ("https://docs.sunpy.org/projects/drms/en/stable/", None),
-    "matplotlib": ("https://matplotlib.org/stable", None),
-    "numpy": ("https://numpy.org/doc/stable/", None),
-    "pandas": ("https://pandas.pydata.org/pandas-docs/stable/", None),
-    "parfive": ("https://parfive.readthedocs.io/en/stable/", None),
-    "python": ("https://docs.python.org/3/", None),
-    "reproject": ("https://reproject.readthedocs.io/en/stable/", None),
-    "scipy": ("https://docs.scipy.org/doc/scipy/reference/", None),
-    "skimage": ("https://scikit-image.org/docs/stable/", None),
-    "sqlalchemy": ("https://docs.sqlalchemy.org/en/latest/", None),
-    "sunpy": ("https://docs.sunpy.org/en/stable/", None),
-}
```

Comment:
this shouldn't be deleted, it might want to be moved, but it should still be there.